Automattic/wp-calypso
# Update how the WordPress.com subdomain is managed from the domains page

#### Changes proposed in this Pull Request

This PR will show remove the free URL from the main domain add a separate entry on the site domains page for the free URL  See: pcYYhz-e4-p2#free-url-domain-item

Note that this PR is focused specifically on the single line at the bottom of the Site Domains page.

Free URL that is not the primary domain for a site:
<img width="1116" alt="Screen Shot 2021-07-19 at 8 52 58 AM" src="https://user-images.githubusercontent.com/1379730/126163003-82903a2d-b617-465e-9bc7-f48a0d3b9830.png">

Free URL that is the primary domain for a site:
<img width="1116" alt="Screen Shot 2021-07-19 at 2 51 28 PM" src="https://user-images.githubusercontent.com/1379730/126211896-54daa62f-6fcb-4259-9167-a4bffa103005.png">

Showing how the manage menu works:

https://user-images.githubusercontent.com/1379730/126092706-7c491d33-7919-449d-b694-23009324945e.mov

#### Testing instructions

Make sure to test this with simple, atomic, and .blog subdomain sites.

Also, check a site that has a domain credit and make sure that the free URL shows below the "Claim your free domain" banner.

<img width="1093" alt="Screen Shot 2021-08-02 at 2 48 20 PM" src="https://user-images.githubusercontent.com/1379730/127909426-41b67f78-c7b3-4b98-922a-a0f9081fc50b.png">
